Source/IntegrationTests/TestFiles/LitTests/LitTest/dafny0/GeneralNewtypeMemberResolution.dfy.expect

```diff
@@ -1,13 +1,12 @@
 GeneralNewtypeMemberResolution.dfy(6,8): Error: mutable fields are allowed only in reference types (consider declaring the field as a 'const')
-GeneralNewtypeMemberResolution.dfy(48,11): Error: method 'PrintMe' is inherited from newtype 'Bool' and is not allowed to be re-declared in newtype 'BoolBool'
```

Comment:
The removal of the branch `trait is not TraitDecl` in `ModuleResolver.cs` is what's causing this error message to disappear. The test is needed.